backend/battle/tests/test_view.py

```diff
@@ -1,15 +1,104 @@
-from django.test import TestCase
 from model_bakery import baker
-from battle.models import PokemonTeam
+from common.utils.tests import TestCaseUtils
+from django.urls import reverse
+from battle.models import Battle, PokemonTeam
 from battle.battles.battle import get_winner_for
 
 
-class GetWinnerTest(TestCase):
+class ListBattlesTest(TestCaseUtils):
     def setUp(self):
         super().setUp()
-        self.creator, self.opponent = baker.make('users.User', _quantity=2)
-        battle = baker.make('battle.Battle', creator=self.creator, opponent=self.opponent)
-        self.team_creator = baker.make('battle.Team', battle=battle, trainer=self.creator)
+        self.battle = baker.make('battle.Battle', creator=self.user)
+        self.battle_from_user = Battle.objects.filter(creator=self.user)
+
+    def test_login_user_can_acess_battle_list(self):
+        response = self.auth_client.get(reverse('battle_list'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_view_returns_empty_list(self):
+        battle = Battle.objects.get(creator=self.user)
+        battle.delete()
+        response = self.auth_client.get(reverse('battle_list'))
+        response_qs = response.context_data.get('battle_list')
+        self.assertCountEqual(response_qs, [])
+
+    def test_view_returns_one_battle_in_list(self):
+        response = self.auth_client.get(reverse('battle_list'))
+        response_qs = response.context_data.get('battle_list')
+        self.assertCountEqual(response_qs, [self.battle_from_user][0])
```

Comment:
Why not do `[self.battle]`?

Reply:
Because I was filtering, but now as you mentioned, I changed to just get method